docs/reference/intro.rst

```diff
@@ -39,51 +52,62 @@ and on Mac OS X under::
+(For details on the D-Wave API for determining platform-independent paths to user
+data and configuration folders see the homebase_ tool.)
 
 .. _homebase: https://github.com/dwavesystems/homebase
 
+A single D-Wave Cloud Client configuration file can contain multiple profiles, each
+defining a separate combination of communication parameters such as the URL to the
+remote resource, authentication token, solver, etc.
+Configuration files conform to a standard Windows INI-style format:
+profiles are defined by sections such as, ``[profile-a]`` and ``[profile-b]``.
+Default values for undefined profile keys are taken from the ``[defaults]`` section.
 
-One config file can contain multiple profiles, each defining a separate
-      (endpoint, token, solver, etc.) combination. Since config file conforms to a
-      standard Windows INI-style format, profiles are defined by sections like:
-      ``[profile-a]`` and ``[profile-b]``.
-
-      Default values for undefined profile keys are taken from the ``[defaults]``
-      section.
-
-      For example, assuming ``~/.config/dwave/dwave.conf`` contains::
+For example, if the configuration file, ``~/.config/dwave/dwave.conf``, selected
+through auto-detection as the default configuration, contains the following
+profiles::
 
           [defaults]
-          endpoint = https://cloud.dwavesys.com/sapi
+          endpoint = https://url.of.some.dwavesystem.com/sapi
           client = qpu
 
           [dw2000]
           solver = DW_2000Q_1
-          token = ...
+          token = ABC-123456789123456789123456789
 
           [software]
           client = sw
           solver = c4-sw_sample
-          token = ...
-
-          [alpha]
-          endpoint = https://url.to.alpha/api
+          token = DEF-987654321987654321987654321
           proxy = http://user:pass@myproxy.com:8080/
-          token = ...
 
-      We can instantiate a client for D-Wave 2000Q QPU endpoint with
+You can instantiate a client for D-Wave 2000Q QPU remote resource with::
 
       >>> from dwave.cloud import Client
-      >>> client = Client.from_config(profile='dw2000')
+      >>> client = Client.from_config(profile='dw2000')   # doctest: +SKIP
 
-      and a client for remote software solver with::
+and a client for a software solver with::
 
-      >>> client = Client.from_config(profile='software')
+      >>> client = Client.from_config(profile='software')   # doctest: +SKIP
 
-      ``alpha`` profile will connect to a pre-release API endpoint via defined HTTP
-      proxy server.
+.. _environmentVariables:
 
+Environment Variables
+---------------------
+
+In addition to D-Wave Cloud Client configuration files, configuration information
+can be set in environment variables; for example:
+
+* ``DWAVE_CONFIG_FILE`` may select the configuration file path.
+* ``DWAVE_PROFILE`` may select the name of a profile (section).
+* ``DWAVE_API_CLIENT`` may select the API client class.
+
```

Comment:
The most useful env var will probably be `DWAVE_API_TOKEN`, so I would list that instead of `DWAVE_API_CLIENT`.

Reply:
Done